File: ReactApp/address-app-react/src/app/models/addresses/AddressSearchResultsModel.ts
import { IAddressSearchResultsModel } from "./interfaces/IAddressSearchResultsModel";

export class AddressSearchResultsModel implements IAddressSearchResultsModel{
    id: string | null = null;
    streetNumber: string | null = null;
    streetName: string | null = null;
    city: string | null = null;
    stateId: string | null = null;
    stateName: string | null = null;
    zipCode: string | null = null;

}

File: BlazorApp/BlazorAddressApp/BlazorAddressApp/AddressApp/Models/Addresses/AddressSearchResultsModel.cs
namespace AddressApp.Models.Locations
{
    public class AddressSearchResultsModel
    {
        public Guid? Id { get; set; }
        public string StreetNumber { get; set; }
        public string StreetName { get; set; }
        public string ApartmentNumber { get; set; }
        public string City { get; set; }
        public string StateName { get; set; }
        public string ZipCode { get; set; }

    }
}
